examples/minimal/.meteor/versions

```diff
@@ -25,16 +25,18 @@ email@2.0.0
 es5-shim@4.8.0
 fetch@0.1.1
 geojson-utils@1.0.10
-hot-code-push@1.0.4
 http@1.4.2
 id-map@1.1.0
 inter-process-messaging@0.1.1
 littledata:synced-cron@1.5.1
-livedata@1.0.18
 logging@1.1.20
 meteor@1.9.3
+<<<<<<< HEAD
+minimongo@1.6.0
+=======
 meteor-base@1.4.0
 minimongo@1.6.1
+>>>>>>> 7581b3f2f1dd6869266d7158f7ba4218d0991897
```

Comment:
It can't possibly work at all with these kind of merge comments in the versions file, please do QA, look at the diffs